server/src/main/java/org/apache/gravitino/server/web/filter/GravitinoInterceptionService.java

```diff
@@ -140,28 +148,23 @@ public Object invoke(MethodInvocation methodInvocation) throws Throwable {
           Map<String, Object> pathParams = extractPathParamsFromParameters(parameters, args);
           AuthorizationExpressionEvaluator authorizationExpressionEvaluator =
               new AuthorizationExpressionEvaluator(expression);
-          boolean authorizeResult =
-              authorizationExpressionEvaluator.evaluate(
-                  metadataContext,
-                  pathParams,
-                  new AuthorizationRequestContext(),
-                  Optional.ofNullable(entityType));
+          AuthorizationRequest.RequestType requestType =
+              extractAuthorizationRequestTypeFromParameters(parameters);
+          executor =
+              switch (requestType) {
+                case COMMON -> new CommonAuthorizerExecutor(
```

Comment:
Could extract these classes to single files?

Reply:
fixed